apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterPreview.tsx

```diff
@@ -113,6 +113,7 @@ const NewsletterPreview: React.FC<{newsletter: Newsletter}> = ({newsletter}) =>
         headerTitle={headerTitle}
         senderEmail={renderSenderEmail()}
         senderName={newsletter.sender_name || title}
+        senderReplyTo={newsletter.sender_reply_to}
```

Comment:
This value can be `newsletter` or `support` too, so to avoid showing those values, we'll need to map it first to the right email address.

You can use the `renderReplyToEmail` method from `apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterDetailModal.tsx` to do this

Reply:
If you implement this, maybe also try to remove the duplicate implementation of `renderSenderEmail`. And use the one from `apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterDetailModal.tsx` instead (this one is more correct)

Reply:
Okay, I did both, I _think_ it went okay, but could use another look from you and @sagzy 🙏